spec/fixtures/files/datahashes/core/collectionobject1.json

```diff
@@ -165,12 +165,12 @@
     "viewersPersonalExperience": "personal",
     "viewersPersonalResponse": "response",
     "viewersReference": "reference;Viewer's;menje",
-    "viewersContributionNote": "Viewer's contribution note",
+    "viewersContributionNote": "Viewer's contribution note\nNext line",
```

Comment:
Data input in record had a newline at the end. This made me realize we'd better test that importing/mapping multiline data works. Updated XML and JSON data to test.